pkg/api/defaults.go

```diff
@@ -76,5 +76,6 @@ func DefaultOperatorFlags() OperatorFlags {
 		"aro.routefix.enabled":                     flagTrue,
 		"aro.storageaccounts.enabled":              flagTrue,
 		"aro.workaround.enabled":                   flagTrue,
+		"aor.autosizednodes.enable":                flagFalse,
```

Comment:
So we are not enabling it for new and existing clusters. What is the plan for rolling this out?

Reply:
First roll out the change, enable for canary region and observe for any potential stability issue. Once this is fine, rollout gradually everywhere.

Reply:
Just noting that there is a typo in the flag name: **aor**.autosizednodes.enable -> **aro**.autosizednodes.enable.

Reply:
Good eye, thank you. Fixed.